subprojects/docs/src/docs/userguide/custom_plugins.adoc

```diff
@@ -154,9 +154,9 @@ Although there are conventional similarities between plugin ids and package name
 [[sec:publishing_your_plugin]]
 ==== Publishing your plugin
 
-If you are publishing your plugin internally for use within your organization, you can publish it like any other code artifact. See the <<publishing_ivy,ivy>> and <<publishing_maven,maven>> chapters on publishing artifacts.
+If you are publishing your plugin internally for use within your organization, you can publish it like any other code artifact. See the <<publishing_ivy,Ivy>> and <<publishing_maven,Maven>> chapters on publishing artifacts.
 
-If you are interested in publishing your plugin to be used by the wider Gradle community, you can publish it to the http://plugins.gradle.org[Gradle plugin portal]. This site provides the ability to search for and gather information about plugins contributed by the Gradle community. See the instructions http://plugins.gradle.org/docs/submit[here] on how to make your plugin available on this site.
+If you are interested in publishing your plugin to be used by the wider Gradle community, you can publish it to the http://plugins.gradle.org[Gradle Plugin Portal]. This site provides the ability to search for and gather information about plugins contributed by the Gradle community. Please refer to the corresponding https://guides.gradle.org/publishing-plugins-to-gradle-plugin-portal/[_guide_] on how to make your plugin available on this site.
```

Comment:
For future reference, [one of my pull requests](https://github.com/gradle/gradle/pull/5783/files#diff-0d3607f604a029ea90520db7d571bcec) adds a `guidesUrl` Asciidoctor attribute. You might want to consider adding it as part of this pull request instead. Then you would be able to use `{guidesUrl}/publishing-plugins-to-gradle-plugin-portal/[_guide_]`.

Also, I don't see why you would use italics for "guide" here. My intention is to use italics for chapter or guide titles, not just any link to a chapter or guide.

Reply:
- [ ] Remove italicisation from "guide" link